adaptivealerting/metric_profiler.py

```diff
@@ -0,0 +1,30 @@
+from statsmodels.tsa.stattools import adfuller
+
+
+def build_profile(x):
+    """
+    Builds a feature profile of the given time series.
+
+    :param x: time series
+    :return: time series feature profile
+    """
+
+    return {
+        "stationary": _is_stationary(x, "1%")
+    }
+
+
+def _is_stationary(x, significance):
+    """
+    Runs a stationarity test on the time series.
+
+    :param x: time series
+    :param significance: valid values are "1%", "5%" and "10%"
+    :return: boolean indicating whether the time series is stationary, assuming the given significance level
+    """
+    result = adfuller(x)
+    test_stat = result[0]
+    p_value = result[1]
+    crit_value = result[4][significance]
+    print("\ncrit_value_{}={}, test_stat={}, p_value={}".format(crit_value, test_stat, significance, p_value))
+    return test_stat < crit_value
```

Comment:
Should this be at the top level of adaptivealerting? Might be better in a subdirectory.

Reply:
I'm pretty new to Python but logically I don't think it makes a difference. It's a top-level module either way. In terms of the physical implementation, I think a file makes more sense right now because I don't see that it makes sense to break any of this out. (The `build_profile` function grows as we add new features, and the corresponding stat tests are few and small enough to simply inline.)